.travis.yml

```diff
@@ -11,8 +11,9 @@ addons:
 language: go
 branches:
   only:
+    - csi-v0.3
     - master
-    - csi-v1.0
+    - csi-v1.0  # remove this once csi-v1.0 becomes master
```

Comment:
We should chat about this (but not block your PR); what I was getting at before was that ideally IMO there would still be a v1.0 branch as the stable latest release and a master that could be at say "v1.0.1" or "v2.0.0" etc.  So a constantly moving up to date master and however many stable release branches.  I realize that currently that's just v0.3.0 and v1.0.0 and we're early enough in 1.0 that we're not worrying much about updating things here.